torchgeo/trainers/utils.py

```diff
@@ -95,3 +97,51 @@ def load_state_dict(model: Module, state_dict: Dict[str, Tensor]) -> Module:
     model.load_state_dict(state_dict, strict=False)  # type: ignore[arg-type]
 
     return model
+
+
+def reinit_initial_conv_layer(
```

Comment:
I think there was some talk on Slack about using timm to do this for us? Is that still something we are considering as an alternative to this?

Reply:
Yes, it was kicked off by thinking about this PR